llumnix/llumlet/request.py

```diff
@@ -33,6 +36,8 @@ def reset_migration_args(self):
         self.last_preemption_time = None
         self.stage_timestamps = []
         self.stage_num_blocks_list = []
+        # By default, there is no limit on the number of steps expected for the request."
+        self.expected_steps = -1
```

Comment:
set to `inf` may better. you can always use `output_len > self.expected_steps ` with out `self.expected_steps > 0`